docs/Schema-Model.md

```diff
@@ -2,136 +2,148 @@
 
 This documents describes the schema modelling for JanusGraph, when alerts are being pushed by Grafink. It also briefly describes the algorithm to be followed for loading the data.
 
-A single alert row has the following fields in hbase
+There are currently 4 column families in the main HBase table:
+- `i`: original alert information,
+- `d`: added values by Fink,
+- `b`: binary data (e.g. binary gzipped FITS image),
+- `a`: for annotations (initially empty).
 
-Note that this schema is subject to evolution.
+A single alert row has the following fields in HBase
 ```
-key: ZTF19acmbtud_2458789.0279514_128.2831039_15.3663548
+key: schema_0.6.0_0.3.2
```

Comment:
This row key must be unique for every alert no?

Reply:
The script that pushes data to the HBase table has two parts:
1. push alert data into HBase table (rowkey `<objectId>_<timestamp>`): https://github.com/astrolabsoftware/fink-broker/blob/92c8988e795e6454602d237ed255e863b48885a1/bin/science_archival.py#L95
2. push a **single schema row** into HBase table (rowkey `schema_<fink broker version>_<fink science version>`): https://github.com/astrolabsoftware/fink-broker/blob/92c8988e795e6454602d237ed255e863b48885a1/bin/science_archival.py#L123

The schema row is a unique row describing data types (unlike other rows that are the data itself).